test_resources/serialization_baseline/databases/My Database/tables/Schemaless Table/fields/Some Field.yaml

```diff
@@ -35,5 +35,9 @@ serdes/meta:
   model: Table
 - id: Some Field
   model: Field
+database_default: null
 database_indexed: null
+database_is_generated: null
```

Comment:
@piranha i would expect we re-sync tables after we import them, right? could be coming from another version.

if so, this feels like dumb noise in our test data and perhaps we can strip out these fields somehow.